crates/scroll/hardforks/src/hardfork.rs

```diff
@@ -107,6 +110,7 @@ impl ScrollHardfork {
                 Self::Curie => Some(7096836),
                 Self::Darwin => Some(8568134),
                 Self::DarwinV2 => Some(8923772),
+                _ => None,
```

Comment:
Should we have `Self::Euclid` and `Self::EuclidV2` as Some(0)?

Reply:
I think that's something TBD?

Reply:
oh right, maybe just add a TODO?